docs/index.rst

```diff
@@ -2,13 +2,28 @@
 Specreduce Documentation
 ########################
 
-The `specreduce <https://specreduce.readthedocs.io/en/latest/index.html>`_ package aims to provide a data reduction toolkit for optical
+The `specreduce <https://specreduce.readthedocs.io/en/latest/index.html>`_ package
+aims to provide a data reduction toolkit for optical
 and infrared spectroscopy, on which applications such as pipeline processes for
 specific instruments can be built. The scope of its functionality is limited to
-basic spectroscopic reduction, with basic *image* processing steps (such as
-bias subtraction) instead covered by `ccdproc <https://ccdproc.readthedocs.io/en/latest/>`_ and other packages, data
-analysis by `specutils <https://specutils.readthedocs.io/en/latest/>`_ and visualization by `matplotlib <https://matplotlib.org/>`_. A few
-examples will nevertheless be provided of its usage in conjunction with these
+basic spectroscopic reduction, currently encompassing the following three tasks:
+
+#. Determining the trace of a spectrum dispersed in a 2D image, either by setting a flat
+   trace or fitting a spline, polynomial, or other model to the positions of the dispersed
```

Comment:
I would also add that users can set a custom trace so `ArrayTrace` isn't left out.